.gitlab/ci/.gitlab-ci.global.yml

```diff
@@ -301,15 +301,15 @@
     reports:
       coverage_report:
         coverage_format: cobertura
-        path: /builds/xsoar/content/artifacts/coverage_report/coverage.xml
+        path: ${CI_PROJECT_DIR}/artifacts/coverage_report/coverage.xml
     expire_in: 30 days
     paths:
-      - /builds/xsoar/content/unit-tests
-      - /builds/xsoar/content/artifacts/*
-      - /builds/xsoar/content/pipeline_jobs_folder/*
+      - ${CI_PROJECT_DIR}/unit-tests
+      - ${CI_PROJECT_DIR}/artifacts/*
+      - ${CI_PROJECT_DIR}/pipeline_jobs_folder/*
     when: always
   services:
-    - name: docker.art.code.pan.run/build-tools--image-dind:20.10.12-dind
+    - name: ${DOCKER_IO}/library/docker:20.10.12-dind
```

Comment:
`DOCKER_IO`, or `DOCKER_HOST`?
(Also in other places.)

Reply:
`DOCKER_IO`..